src/biokbase/narrative/tests/test_jobcomm.py

```diff
@@ -244,7 +322,7 @@ def test_lookup_job_info_no_job(self):
     def test_lookup_job_info_bad_job(self):
         job_id = "nope"
         req = make_comm_msg("job_info", job_id, True)
-        with self.assertRaises(ValueError) as e:
+        with self.assertRaises(NoJobException) as e:
```

Comment:
Have to change these since `assertRaises` does not factor in exceptions subclassing each other

Reply:
Can you convert these to `assertRaisesRegex` tests with the additional check of the error message?